pyarlo/camera.py

```diff
@@ -114,7 +118,6 @@ def captured_today(self):
         """Return list of <ArloVideo> object captured today."""
         return self.videos(days=0)
 
-    @property
     def play_last_video(self):
```

Comment:
**BREAKING:** this felt like more of an action than a property, so i made it a method.

Reply:
+1